### PR TITLE
cuda.py changed  to acommodate version 10.1

### PR DESCRIPTION
**Pull request by sassy-crick**

### CODE REVIEW

File: easybuild/easyblocks/c/cuda.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ##
 # This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
 #
-# Copyright:: Copyright 2012-2019 Cyprus Institute / CaSToRC, Uni.Lu, NTUA, Ghent University, Forschungszentrum Juelich GmbH
+# Copyright:: Copyright 2012-2018 Cyprus Institute / CaSToRC, Uni.Lu, NTUA, Ghent University, Forschungszentrum Juelich GmbH
```
[the remark this answers is not in the document]
Comment (wpoely86) [reply easybuild/easyblocks/c/cuda.py]: Euh, why change this back to 2018?

[the remark this answers is not in the document]
Comment (sassy-crick) [reply easybuild/easyblocks/c/cuda.py]: left as that is a comment line.

File: easybuild/easyblocks/c/cuda.py [hunk 2]
```diff
@@ -69,15 +69,24 @@ def install_step(self):
...
+            install_interpreter = "perl"
             install_script = "cuda-installer.pl"
             # note: also including samples (via "-samplespath=%(installdir)s -samples") would require libglut
             self.cfg.update('installopts', "-verbose -silent -toolkitpath=%s -toolkit" % self.installdir)
+        else:
+            install_interpreter = ""
+            install_script = "./cuda-installer"
+            # note: also including samples (via "-samplespath=%(installdir)s -samples") would require libglut
+            self.cfg.update('installopts', "--silent --toolkit --toolkitpath=%s --defaultroot=%s" % (
+            self.installdir, self.installdir))
```
[the remark this answers is not in the document]
Comment (wpoely86) [reply easybuild/easyblocks/c/cuda.py]: I meant:

```
self.cfg.update('installopts', "--silent --toolkit --toolkitpath=%s --defaultroot=%s" % (
                self.installdir, self.installdir))
```

[the remark this answers is not in the document]
Comment (sassy-crick) [reply easybuild/easyblocks/c/cuda.py]: Thanks. Got lost in translation.

**Comment by boegel** [comment]
Thanks a lot for tackling this @sassy-crick!

**Pull request merged**